setup.py

```diff
@@ -28,12 +28,11 @@ def __init__(self, name, sourcedir=""):
 class CMakeBuild(build_ext):
     def build_extension(self, ext):
         extdir = os.path.abspath(os.path.dirname(self.get_ext_fullpath(ext.name)))
-
-        # required for auto-detection of auxiliary "native" libs
```

Comment:
Do you want to remove the comment? I found this a bit surprising why this is needed, so I added it.

Reply:
I was looking at a Python 3 only version, which has the end separator automatically (due to pathlib) and has less need of the comment. Don't think I really intended to remove the comment here.